### DISCUSSION TITLE
Should seniority be nullable on StockClass.schema.json ?

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#239] [General]
Again since all systems might not have this data (it's a nullable field in Shareworks for example) I can see people putting in 0 or some other number when really it would just be more clear with null

**Comment by JSv4** [comment]
Hey @jacobyavis, how do you treat nulls in your system when you're looking at seniority? We were discussing this today and think that everything should have some seniority value. If no security is senior to another, they'd all have the same seniority. If you store something as null, does that basically mean it's lowest (or highest seniority)?
